.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.6
```

Comment:
Any reason this file is here?

Reply:
It's only for the development of the gem, so I usually leave one to indicate which version of ruby the test/dev dependencies are known to work against.

But I deleted it, as requested.